docs/reference/migration/migrate_2_1.asciidoc

```diff
@@ -24,4 +24,4 @@ GET /my_index/_search?scroll=2m
 
 Scroll requests sorted by `_doc` have been optimized to more efficiently resume
 from where the previous request stopped, so this will have the same performance
-characteristics as the former `scan` search type.
```

Comment:
Add something here about renaming position_offset_gap to position_increment_gap.